### ISSUE TITLE
Shouldn't swipe-js-iso be a dependency?

### ISSUE THREAD

**Issue by namuol** [voronianski/react-swipe#73]
Since it's absolutely required to function, `swipe-js-iso` should be declared as a dependency in `package.json`. Is there any particular reason it's declared as a devDependency, but not a peerDependency or ordinary dependency?

**Comment by vishy1618** [comment]
Hello, are we planning to release a new version with this fix any time soon?

**Comment by voronianski** [comment]
@vishy1618 released `5.0.4`

**Comment by vishy1618** [comment]
@voronianski I think there still might be an issue. The `package.json` still has a redundant `dependencies` section at the bottom that is blank:

``` json
{
  "name": "react-swipe",
  "version": "5.0.4",
  "description": "Brad Birdsall's Swipe.js, as a React component",
  "homepage": "https://github.com/voronianski/react-swipe",
  "keywords": [
    "react",
    "component",
    "carousel",
    "swipe",
    "react-component"
  ],
  "author": "Dmitri Voronianski <dmitri.voronianski@gmail.com>",
  "contributors": [
    "Jed Schmidt <where@jed.is>"
  ],
  "repository": {
    "type": "git",
    "url": "git://github.com/voronianski/react-swipe.git"
  },
  "main": "index.js",
  "license": "MIT",
  "dependencies": {
    "swipe-js-iso": "^2.0.3"
  },
  "devDependencies": {
    "ava": "^0.13.0",
....
  },
  "dependencies": {}
}
```

This breaks modules systems that don't have an implicit peer dependency like Brunch for example.
